src/lib/api/index.ts

```diff
@@ -2,57 +2,57 @@
+export type { ForgeInternalErrorResponse } from "./models/ForgeInternalErrorResponse";
+export type { ForgeInvalidUploadResponse } from "./models/ForgeInvalidUploadResponse";
+export type { ForgeValueErrorResponse } from "./models/ForgeValueErrorResponse";
+export type { GnarkCircuitInfoResponse } from "./models/GnarkCircuitInfoResponse";
+export type { Halo2CircuitInfoResponse } from "./models/Halo2CircuitInfoResponse";
+export type { JobStatus } from "./models/JobStatus";
+export type { JWTTokenErrorResponse } from "./models/JWTTokenErrorResponse";
+export type { NoirCircuitInfoResponse } from "./models/NoirCircuitInfoResponse";
+export type { ObtainApikeyInput } from "./models/ObtainApikeyInput";
+export type { PagedProofInfoResponse } from "./models/PagedProofInfoResponse";
+export type { Plonky2CircuitInfoResponse } from "./models/Plonky2CircuitInfoResponse";
+export type { ProjectDoesNotExistResponse } from "./models/ProjectDoesNotExistResponse";
+export type { ProjectInfoResponse } from "./models/ProjectInfoResponse";
+export type { ProjectLatestCircuitResponse } from "./models/ProjectLatestCircuitResponse";
+export type { ProofCannotBeCreatedResponse } from "./models/ProofCannotBeCreatedResponse";
+export type { ProofDoesNotExistResponse } from "./models/ProofDoesNotExistResponse";
+export type { ProofInfoResponse } from "./models/ProofInfoResponse";
+export type { Schema } from "./models/Schema";
+export type { SmartContractVerifierResponse } from "./models/SmartContractVerifierResponse";
+export type { TeamDetail } from "./models/TeamDetail";
+export type { TeamDoesNotExistResponse } from "./models/TeamDoesNotExistResponse";
+export type { TeamMeResponse } from "./models/TeamMeResponse";
+export type { TokenObtainPairInputSchema } from "./models/TokenObtainPairInputSchema";
+export type { TokenObtainPairOutputSchema } from "./models/TokenObtainPairOutputSchema";
+export type { TokenRefreshInputSchema } from "./models/TokenRefreshInputSchema";
+export type { TokenRefreshOutputSchema } from "./models/TokenRefreshOutputSchema";
+export type { TokenVerifyInputSchema } from "./models/TokenVerifyInputSchema";
+export type { UserMeResponse } from "./models/UserMeResponse";
+export type { ValidationErrorResponse } from "./models/ValidationErrorResponse";
 
-export { AuthorizationService } from './services/AuthorizationService';
-export { CircuitsService } from './services/CircuitsService';
-export { InternalService } from './services/InternalService';
-export { ProofsService } from './services/ProofsService';
-export { TokenService } from './services/TokenService';
+export { AuthorizationService } from "./services/AuthorizationService";
+export { CircuitsService } from "./services/CircuitsService";
+export { InternalService } from "./services/InternalService";
+export { ProofsService } from "./services/ProofsService";
+export { TokenService } from "./services/TokenService";
```

Comment:
It was a minor bug that these files directly under `src/lib/api/` weren't being formatted with prettier. I updated that so that the code involved with diffs/patches would always be formatted.